backend/api/Services/MissionLoaders/MissionLoaderInterface.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Api.Controllers.Models;
+using Api.Database.Models;
+namespace Api.Services.MissionLoaders
+{
+    public interface IMissionLoader
+    {
+        public Task<MissionDefinition?> GetMissionById(string sourceMissionId);
+
+        public Task<IQueryable<MissionDefinition>> GetAvailableMissions(string? installationCode);
+
+        public Task<List<MissionTask>> GetTasksForMission(string sourceMissionId);
+
+        public Task<List<PlantInfo>> GetPlantInfos(); // Facility service
```

Comment:
I am not sure if this comment is very helpful right now

Reply:
Probably not, ill remove it =)